tests/components/conftest.py

```diff
@@ -624,7 +624,8 @@ async def _validate_translation(
     if not translation_required:
         return
 
-    if full_key in translation_errors:
+    if translation_errors.get(full_key) == "unused":
+        # This translation key is in the ignore list, mark it as used
```

Comment:
Without this check, we would overwrite the error for a missing translation which was accessed more than once with "used"